### PR TITLE
catalog was created

### PR DESCRIPTION
**Pull request by BilokonYaroslav**
- [DEMO LINK](https://BilokonYaroslav.github.io/layout_catalog/)
- [TEST REPORT LINK](https://BilokonYaroslav.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,451 @@
...
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#item" class="header__link">Samsung</a>
+          </li>
+          <li class="header__item">
+            <a href="#item" class="header__link">Smartphones</a>
+          </li>
+
```
Comment (IvanFesenko) [inline src/index.html:30]:
```suggestion
          <li class="header__item">
            <a href="#item" class="header__link">Samsung</a>
          </li>
          
          <li class="header__item">
            <a href="#item" class="header__link">Smartphones</a>
          </li>

```

File: src/styles/blocks/header.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  &__link-img {
+    margin: 10px 0 10px 50px;
+    height: 40px;
+  }
+
+  &__img {
+    width: 40px;
+    height: 40px;
+  }
```
Comment (IvanFesenko) [inline src/styles/blocks/header.scss:16]: Redudant

File: src/styles/blocks/header.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+
+  &__link-img {
+    margin: 10px 0 10px 50px;
```
Comment (IvanFesenko) [inline src/styles/blocks/header.scss:9]: Use padding for header instead margin

File: src/index.html [hunk 4]
```diff
@@ -7,6 +7,452 @@
...
+    <header class="header">
+      <a href="#home" class="header__link-img">
+        <img src="images/logo.png"
+          alt="MOYO__logo"
+          class="header__img">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#item" class="header__link header__link--active is-active">
```
Comment (YegorKorenkov) [inline src/index.html:19]: Please remove is-active class, title does not follow BEM, and I don't see where u use it in styles

**Review by YegorKorenkov — COMMENTED** [review]
One remark, but everything okey. Nice job!

File: src/index.html (hunk 4 above)
Comment (BilokonYaroslav) [reply src/index.html:19]: - add class `is-active` to the first link (`Apple`) in navigation
from the readme file

Comment (BilokonYaroslav) [reply src/index.html:19]: I think ше'і for test

Comment (BilokonYaroslav) [reply src/index.html:19]: I think it`s for tests

**Review by IvanFesenko — APPROVED** [review]

**Pull request closed without merging**